src/TSTransformer/nodes/expressions/transformCallExpression.ts

```diff
@@ -66,6 +66,14 @@ function runCallMacro(
 	return wrapReturnIfLuaTuple(state, node, macro(state, node as never, expression, args));
 }
 
+function isInsideRoactComponent(state: TransformState, node: ts.Node) {
+	const classLikeAncestor = getAncestor(node, ts.isClassLike);
+	if (classLikeAncestor) {
```

Comment:
It took me a while to realize what this meant. Might I suggest `hasAncestorThatIsClassLike`?

Reply:
turns out this isn't a boolean :/ Another reason I don't like truthiness.